docs/installation-and-operation/start.md

```diff
@@ -8,30 +8,15 @@ The birth, care, and feeding of your Metabase.
 
 ## [Installing Metabase](./installing-metabase.md)
```

Comment:
Small comment, if we have added the link down, is it better to remove the link from the title?